pallets/gear/src/internal.rs

```diff
@@ -561,6 +564,12 @@ where
         let from = <T::AccountId as Origin>::from_origin(dispatch.source().into_origin());
         let value = dispatch.value().unique_saturated_into();
 
+        let gas_for_delay = (delay as u64)
+            .saturating_add(CostsPerBlockOf::<T>::reserve_for().unique_saturated_into())
+            .saturating_mul(CostsPerBlockOf::<T>::dispatch_stash().unique_saturated_into());
+
+        let interval_finish;
```

Comment:
better to return value from if-else block

Reply:
done